src/wavm/WAVMWasmModule.cpp

```diff
@@ -188,11 +184,14 @@ void WAVMWasmModule::clone(const WAVMWasmModule& other)
 
         // Remap dynamic modules
         // TODO - double check this works
+        lastLoadedDynamicModuleHandle = other.lastLoadedDynamicModuleHandle;
```

Comment:
This `lastLoadedDynamicModuleHandle` is unfortunately needed for linking (we load the module, then some time later WAVM tries to link it, at which point we need to revisit the thing we just created. This happens in the `resolve` method).

Reply:
I understand this actually works now right? Could we remove the `TODO`?

Reply:
Good point, yes I'm pretty sure this is covered by a test somewhere